### PR TITLE
✨ feat: SQ-65079 [Frontend] Criar componente de gráfico para gênero e idade

### PR DESCRIPTION
**Pull request by wandersonsales-dev**
Card: [SQ-65079](https://duopana.myjetbrains.com/youtrack/issue/SQ-65079/Frontend-Criar-componente-de-grafico-para-genero-e-idade)

## Summary by CodeRabbit

- **Bug Fixes**
	- Improved tooltip content rendering by modifying how text is added, enhancing display consistency.
- **New Features**
	- Enhanced UI component functionality for smoother transitions and interactions.
	- Optimized color management and token structures for improved readability and maintainability.

### CODE REVIEW

**Comment by wandersonsales-dev** [comment]
@JonasPeres code review

File: src/js/components/tooltip.js [hunk 1]
```diff
@@ -38,7 +38,7 @@
     e.target.setAttribute('data-tooltip-open', tooltipCount)
     tooltipArrow.className = 'tooltip-arrow'
     tooltip.appendChild(tooltipArrow)
-    tooltip.append(e.target.getAttribute('data-tooltip'))
+    tooltip.innerHTML += (e.target.getAttribute('data-tooltip'))
```
[the remark this answers is not in the document]
Comment (wandersonsales-dev) [reply src/js/components/tooltip.js:41]: It's necessary

**Review by elianefaveron1 — DISMISSED** [review]

**Review by htamagnus — DISMISSED** [review]

**Review by elianefaveron1 — DISMISSED** [review]

**Comment by wandersonsales-dev** [comment]
@JonasPeres code review

**Comment by wandersonsales-dev** [comment]
@JonasPeres code review

**Review by htamagnus — APPROVED** [review]

**Review by JonasPeres — APPROVED** [review]

**Pull request merged**